iOS-13-Migration-Guide.md

```diff
@@ -45,6 +45,39 @@ This section provides information required for existings apps built with Xcode 1
     Error Domain=com.twilio.voice.error Code=31301 "Http status: 400. Unexpected registration response." UserInfo={NSLocalizedDescription=Http status: 400. Unexpected registration response.}
     ```
 
+    If you are using the Voice iOS SDK 5.3.0 and above, pass the credential data as the argument directly to the `[TwilioVoice registerWithAccessToken:deviceTokenData:completion:]` method.
```

Comment:
As mentioned in another comment, we can just talk about the latest. Thoughts?

Reply:
+1.  

Gave some thoughts after our discussion, the migration guide doesn't need to mention 2.x to <5.3.0 separately.  

The only drawback is if some customers have not followed the migration guide with 5.2.x SDK, they may want to see it - but in that case, our recommendation is to use the latest (5.3.0) sdk.